modules/ROOT/pages/use-log4j-appender.adoc

```diff
@@ -96,6 +96,21 @@ For the latest version of Splunk, see https://github.com/splunk/splunk-library-j
 
 After your application starts, logs start flowing to your custom Log4j appender, and you can view them on your target logging system.
 
+== Disable Custom Log4j
+
+If you enabled Custom Log4j in the `values.yaml` during the installation of Runtime Fabric and you want to disable it the RTF cluster level, follow these steps:
```

Comment:
```suggestion
If you enabled Custom Log4j in the `values.yaml` during the installation of Runtime Fabric and you want to disable it at the Runtime Fabric cluster level, follow these steps:
```

Reply:
We aren't ever supposed to abbreviate Runtime Fabric to RTF in user documentation.